scripts/app-stream/start-image-builder.js

```diff
@@ -0,0 +1,97 @@
+
+const { AppStreamClient, DescribeImageBuildersCommand, CreateImageBuilderCommand } = require("@aws-sdk/client-appstream");
+const { DescribeVpcsCommand, EC2Client, DescribeSubnetsCommand} = require("@aws-sdk/client-ec2");
+
+class StartImageBuilder {
+    constructor(profile, region) {
+        console.log(`Starting Image Builder using AWS Profile ${profile} in region ${region}`);
+        this.appStreamClient = new AppStreamClient({ region, profile});
+        this.region = region;
+        this.ec2Client = new EC2Client({region, profile});
+        this.imageBuilderName = `SWBImageBuilder-${Date.now()}`;
+    }
+
+    async run() {
+        try {
+            await this.createImageBuilder();
+            await this.waitForImageBuilderToBeReady();
+
+            const imageBuilderUrl = `https://console.aws.amazon.com/appstream2/home?region=${this.region}#/images?bottomTab=details&topTab=image-builders`
+
+            console.log(`You can find your new Image Builder at this address ${imageBuilderUrl}`)
```

Comment:
log the image builder name too so that admins can locate it easily?

Reply:
Added